library/Exceptions/UniqueException.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+ * This file is part of Respect/Validation.
+ *
+ * (c) Alexandre Gomes Gaigalas <alexandre@gaigalas.net>
+ *
+ * For the full copyright and license information, please view the "LICENSE.md"
+ * file that was distributed with this source code.
+ */
+
+namespace Respect\Validation\Exceptions;
+
+class UniqueException extends ValidationException
+{
+    public static $defaultTemplates = [
+        self::MODE_DEFAULT => [
+            self::STANDARD => '{{name}} must not contain duplicates',
+        ],
+        self::MODE_NEGATIVE => [
+            self::STANDARD => '{{name}} must contain at least one duplicate',
```

Comment:
I think this is too specific, how about _{{name}} must contain duplicates_?

***
Not a change request here